projects/refer-and-monitor-and-delius/src/test/kotlin/uk/gov/justice/digital/hmpps/service/NsiServiceTest.kt

```diff
@@ -23,7 +23,9 @@ import uk.gov.justice.digital.hmpps.integrations.delius.referral.NsiStatusHistor
 import uk.gov.justice.digital.hmpps.integrations.delius.referral.NsiStatusRepository
 import uk.gov.justice.digital.hmpps.integrations.delius.referral.entity.NsiStatus
 import uk.gov.justice.digital.hmpps.telemetry.TelemetryService
+import java.time.LocalDate
```

Comment:
Are these imports used?

Reply:
Shame we can't auto-fix these in the CLI: https://youtrack.jetbrains.com/issue/IDEA-242015/Optimize-Imports-via-CLI